app.config.js

```diff
@@ -191,7 +191,7 @@ module.exports = function (config) {
           'expo-build-properties',
           {
             ios: {
-              deploymentTarget: '14.0',
+              deploymentTarget: '15.1',
```

Comment:
Upstream has bumped deployment target to 15.1. We don't have any users on iOS 14 anyway so this is nbd.